### ISSUE TITLE
Folderdiff Filter

### ISSUE THREAD

**Issue by anandpeculiar2017** [MicrosoftDocs/vsts-docs#353]
The Syntax provided for the Filter is not working as expected.(.cs;!objd;!obj;!bin\), please provide the proper syntax to filter the particular extension from only selected folder

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 91277ad3-c7f1-2970-586d-a7268874c413
* Version Independent ID: 421be179-93fe-6c53-0807-f0cbc5405c39
* Content: [Folder comparison filters](https://docs.microsoft.com/en-us/vsts/tfvc/folder-comparison-filters)
* Content Source: [docs/tfvc/folder-comparison-filters.md](https://github.com/MicrosoftDocs/vsts-docs/blob/live/docs/tfvc/folder-comparison-filters.md)
* Service: **unspecified**
* GitHub Login: @douge
* Microsoft Alias: **sdanie**

**Comment by steved0x** [comment]
@anandpeculiar2017 It looks like there are some formatting issues in that article that happened when we migrated from visualstudio.com over to docs.microsoft.com - the filter that you list in your example should be: `*.cs;!objd\;!obj\;!bin\` - it looks like the \* character and some of the \\ characters were being treated as markdown syntax by our doc build system. I've created a work item to get this fix up and sweep the remainder of these docs to make sure there aren't other instances, I think it is mostly limited to command related documents but I'm going to check then all out. This doc should be fixed by today.
